src/RSCommand.cc

```diff
@@ -246,12 +246,18 @@ int FindEntity::construct_protobuf(
 {
     const Json::Value& cmd = jsoncmd[_cmd_name];
 
+    Json::Value results = get_value<Json::Value>(cmd, "results");
```

Comment:
Does this work even when no results block is given with FindEntity?

Reply:
yes, same as the properties above. it will return an empty object, we check for that in PMGDQuery